fonts.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../font/PTSerif-Regular.woff2') format('woff2'),
+    url('../font/PTSerif-Regular.woff') format('woff'),
+    url('../font/PTSerif-Regular.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../font/PTSerif-Bold.woff2') format('woff2'),
+    url('../font/PTSerif-Bold.woff') format('woff'),
+    url('../font/PTSerif-Bold.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Museo Sans Cyrl 500';
+  src: url('../font/MuseoSansCyrl-500.woff2') format('woff2'),
+    url('../font/MuseoSansCyrl-500.woff') format('woff'),
+    url('../font/MuseoSansCyrl-500.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5764748)